app/src/main/java/com/example/consolecalculator/Calculator.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.consolecalculator
+
+import java.util.*
+
+class Calculator {
+
+    fun rpnToAnswer(rpn: String): Double? {
+        try {
+            var operand = String()
```

Comment:
```suggestion
            var operand = ""
```
либо можешь вообще без начального значения указать `var operand: String`